server/audit/views.py

```diff
@@ -64,26 +82,49 @@ def progression_metrics(self, request): # pylint: disable=no-self-use
             status=status.HTTP_200_OK)
 
 
+def fix_audit(dictionary):
+    person = dictionary.pop('initiated_by')
+    dictionary['initiated_by'] = person['first_name'] + ' ' + person['last_name']
+    dictionary['location'] = person['location']
+    time = datetime.strftime(datetime.strptime(dictionary['initiated_on'],
+                                               '%Y-%m-%dT%H:%M:%S.%fZ'), '%d/%m/%Y')
+    dictionary['initiated_on'] = time
+    items = dictionary.pop('inventory_items')
+    dictionary['bin'] = None
+    for item in items:
+        popped_bin = item.pop('Bin')
+        if not dictionary['bin']:
+            dictionary['bin'] = popped_bin
+        else:
+            if popped_bin != dictionary['bin']:
+                dictionary['bin'] = 'Multiple'
+                break
+    return dictionary
+
```

Comment:
I don't get what the use of all this is

Reply:
This formats the data properly for the client side to display the information as requested in the mockups rather then sending an excess of unnecessary data and having the client filter through them

Reply:
awesome, okay, I see